metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -317,6 +316,8 @@ This package installs the Metron Profiler %{metron_home}
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 %changelog
+* Thu Jan 19 2017 Justin Leet <justinjleet@gmail.com> - 0.3.0
```

Comment:
I'm not even sure who we should talk to about how it should be handled.  I'm more than happy to adjust the changelog as needed, but I need to know how to change it first.